app/views/controllers/projects/project_aliases/_form.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= form_with(model: project_alias, url:) do |form| %>
+  <% if project_alias.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(project_alias.errors.count, "error") %> prohibited this project alias from being saved:</h2>
+      <ul>
+        <% project_alias.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= form.label :name %>
+    <%= form.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= form.label :project_id %>
+    <%= form.collection_select :project_id, Project.all, :id, :title %>
+  </div>
+
+  <div class="field">
+    <%= form.label :target_type %>
+    <%= form.select :target_type, ['Location', 'User'] %>
+  </div>
+
+  <div class="field">
+    <%= form.label :target_id %>
+    <%= form.number_field :target_id %>
```

Comment:
For style consistency, remember these form helpers are just Ruby methods, and even though Rubocop doesn't lint .erb (yet) the MO style in all cases, including erb, is parentheses for method args.

Reply:
This was a side effect of using a generator.  Not intentional.  Thanks for catching it.

Reply:
I figured... Just now i got curious if we can configure our generators to follow our rubocop config, but i haven't found a way yet. That would be convenient, right? Maybe I'm not using the right keywords.

Reply:
Yeah, that would be great.  Did you see this: https://guides.rubyonrails.org/generators.html ?